src/main/java/com/icthh/xm/ms/dashboard/web/rest/BulkDashboardResource.java

```diff
@@ -0,0 +1,105 @@
+package com.icthh.xm.ms.dashboard.web.rest;
+
+import com.codahale.metrics.annotation.Timed;
+import com.icthh.xm.ms.dashboard.service.bulk.AtomicBulkDashboardService;
+import com.icthh.xm.ms.dashboard.service.bulk.BulkDashboardService;
+import com.icthh.xm.ms.dashboard.service.dto.BulkDashboardItemStatus;
+import com.icthh.xm.ms.dashboard.service.dto.DashboardDto;
+import lombok.RequiredArgsConstructor;
+import org.springframework.security.access.prepost.PreAuthorize;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+import javax.validation.Valid;
+import java.util.ArrayList;
+import java.util.Collection;
+
+/**
+ * REST controller for managing bulk atomic and non atomic operations with Dashboard resource.
+ */
+@RestController
+@RequiredArgsConstructor
+@RequestMapping("/api/bulk")
+public class BulkDashboardResource {
```

Comment:
Suggest to rename as `DashboardBulkResource`

Reply:
Why ?